ethpm_types/contract_type.py

```diff
@@ -290,6 +290,12 @@ class ContractType(BaseModel):
     **NOTE**: This is not part of the canonical EIP-2678 spec.
     """
 
+    pcmap: Optional[dict] = None
```

Comment:
Where's the link to documentation showing how this functions?

Reply:
There isn't any in Vyper, I only found out the values by reading the Vyper source [here](https://github.com/vyperlang/vyper/blob/master/vyper/ir/compile_ir.py#L739-L753)

Google example: https://www.google.com/search?q=vyper+"pc_pos_map"

Since this isn't in Solidity there wasn't anything to link to there either, unlike for source-map. Should we just detail the format here?

Reply:
it might help to make it somewhat more self-documenting and type-checked using an actual pydantic model, like with SourceMap

Reply:
@NotPeopling2day @fubuloubu just updated in 23ac533 to add a bit more documentation and add a Pydantic model

Now you can do this:

```python
>>> contract_type.pcmap.parse()
{54: PCMapItem(...)}
```

Which will clean up some of my ape-core reverts code since we don't have to deal with numeric strings as keys anymore 👍